src/DiscordMember.js

```diff
@@ -272,7 +272,7 @@ class DiscordMember {
         freshData = true
 
         // Cache the data for future use.
-        await Cache.set('users', this.id, data)
+        await Cache.set('users', this.id, data, false)
```

Comment:
You shouldn't have to specify the TTL for set calls. Instead, let us specify a TTL in the `Cache` per-collection. Not per-key.

You should leave everything else the same, just have a constant object at the top of the file with a list of collections to not clear. Clear any collection not in that object the standard way.